### PR TITLE
user auth

### PR DESCRIPTION
**Pull request by aboubakar20**

### CODE REVIEW

File: src/components/Login.vue [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+      email: "",
+      password: "",
+    };
+  },
+  methods: {
+    login: function() {
+      firebase
+        .auth()
+        .signInWithEmailAndPassword(this.email, this.password)
+        .then(
```
Comment (junaidrt) [inline src/components/Login.vue]: @aboubakar20 Please use `async` `await` instead of `then`.

**Review by junaidrt — CHANGES_REQUESTED** [review]

File: src/components/Login.vue [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+export default {
+  name: "login",
+  data() {
+    return {
+      email: "",
+      password: "",
+    };
+  },
+  methods: {
+    // login: function() {
```
Comment (junaidrt) [inline src/components/Login.vue]: @aboubakar20 Please remove commented out code.

File: src/components/SignUp.vue [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+export default {
+  name: "signUp",
+  data() {
+    return {
+      email: "",
+      password: "",
+    };
+  },
+  methods: {
+    signUp: function() {
```
Comment (junaidrt) [inline src/components/SignUp.vue]: @aboubakar20 Please use a consistent syntax throughout the project. Either use `signup: function(){}` or `signup(){}`.

**Review by junaidrt — CHANGES_REQUESTED** [review]

File: src/components/SignUp.vue (hunk 3 above)
Comment (aboubakar20) [reply src/components/SignUp.vue]: Done

File: src/components/Login.vue (hunk 2 above)
Comment (aboubakar20) [reply src/components/Login.vue]: Done

File: src/components/Login.vue (hunk 1 above)
Comment (aboubakar20) [reply src/components/Login.vue]: Done

**Review by junaidrt — APPROVED** [review]

**Pull request merged**